docs/quickstart.md

```diff
@@ -0,0 +1,65 @@
+## Setup
+
+
+To get started, clone the repo, start a Python environment, and install all dependencies.
+
+``` bash
+git clone git@github.com:fetchai/agents-aea.git
```

Comment:
please replace with the new approach:

pip install aea

Reply:
this installs the basics

for the cli one needs to do: `pip install aea[cli]` and for the whole package `pip install aea[all]`